### ISSUE TITLE
How to use DDPG when installing from git?

### ISSUE THREAD

**Issue by tirafesi** [hill-a/stable-baselines#741]
How can I use DDPG when installing stable-baselines from git?

```bash
> pip install git+https://github.com/hill-a/stable-baselines
> python
>>> from stable_baselines import DDPG
ImportError: cannot import name 'DDPG'
```

**Comment by araffin** [comment]
Hello,

Did you try with the [mpi] option as mentioned in the [documentation](https://stable-baselines.readthedocs.io/en/master/guide/install.html)?

btw, we just did a release so `pip install stable-baselines[mpi] --upgrade` will give you the latest version.

**Comment by tirafesi** [comment]
Wouldn't that defeat the purpose of installing through git? (i.e. getting the latest code)

The reason I was installing through `pip+git` was because I wanted to use the latest callbacks - are those included in the latest pip release?

**Comment by Miffyli** [comment]
We just released the [newest version 2.10.0](https://github.com/hill-a/stable-baselines/releases/tag/v2.10.0) on PyPi, too, so you can now get those callbacks by installing with `pip`. In any case, use the [mpi] option as arrafin said. If that works, you can close this issue.

**Comment by tirafesi** [comment]
Yes, that worked. Thanks!
